main/preferences.js

```diff
@@ -52,9 +52,13 @@ const openPrefsWindow = async options => {
   }
 
   ipc.callRenderer(prefsWindow, 'mount');
-  await pEvent(ipcMain, 'preferences-ready');
```

Comment:
Why not just `await pEvent(ipc.answerRenderer, 'preferences-ready');` ?

Reply:
Wait, is `answerRenderer` also an event emitter? Not just a function that accepts channel etc? Only reason I removed the `pEvent` was because it works with event emitters

Reply:
I could try `await pEvent(ipc, eventName)` but I'd have to use the util to generate the event name the way it's done in electron-better-ipc

Reply:
I was confused. I meant that it could be promisified with `util.promisify`.

Reply:
Ah, makes sense: 2146d1f